packages/shared/lib/core/account/utils/buildEmptyAccountBalance.ts

```diff
@@ -0,0 +1,16 @@
+import { IAccountBalance } from '@core/account'
+
+export function buildEmptyAccountBalance(): IAccountBalance {
```

Comment:
Where is this used?

Reply:
Might be left behind from the earlier implementation of this. Will remove it